Automattic/simplenote-macos
# Editor: Copy Interlink Action

### Details:
In this PR we're adding a new Editor Action, which allows you to **copy a Note's Internal Link** 🔗.

**Plus:** Since I was there, few missing localized strings were added.

[iOS Sibling PR here](https://github.com/Automattic/simplenote-ios/pull/864)
Ref. #663

@guarani Sir!! can I bug you agaaain?
Thank you Paul!!!


### Test: Single Selection
1. Select a single note
2. Click over the top right `( ... )` icon
3. Click over `Copy Internal Link`

- [ ] Verify that you can paste a Markdown Link to the selected note
- [ ] Verify that the Markdown Link looks as follows: `[Title](simplenote://note/HASH)`

### Test: Multiple Selection
1. Select multiple notes with CMD + Click
2. Click over the top right `( ... )` icon

- [ ] Verify that the action `Copy Internal Link` is disabled

### Release
`RELEASE-NOTES.txt` was updated in 7ee5dd3 with:
 
> Editor's Actions now lets you copy a Note's Internal Link
